tests/web3js/debug_traces_test.js

```diff
@@ -283,15 +283,15 @@ it('should retrieve transaction traces', async () => {
         txTrace,
         {
             from: conf.eoa.address.toLowerCase(),
-            gas: '0xb50d',
+            gas: '0xb56b',
```

Comment:
Change comes from: `EIP-7623: Increase calldata cost`.
See: https://mixbytes.io/blog/the-prague-electra-pectra-hardfork-explained#rec860520137 .